pages/news.vue

```diff
@@ -1,6 +1,17 @@
 <template>
   <v-container>
-    <v-row>
+    <v-row> 
+      <v-sheet v-if="notLoaded" class="mx-3">
```

Comment:
```suggestion
      <v-sheet v-if="!loaded" class="mx-3">
```
You can use the `!` (not) syntax instead, and do away with `notLoaded`.